pkg/render/applicationlayer/applicationlayer.go

```diff
@@ -361,6 +361,7 @@ func (c *component) proxyEnv() []corev1.EnvVar {
 		// envoy needs to run as root to be able to use transparent flag (for tproxy)
 		{Name: "ENVOY_UID", Value: "0"},
 		{Name: "ENVOY_GID", Value: "0"},
+		{Name: "TIGERA_TPROXY_ENABLED", Value: ""},
```

Comment:
I think we should be more explicit and having a value of true or false specified or I'd like to see the name be TIGERA_TPROXY and the values of Enabled or Disabled.

Reply:
This one follows as other envoy env vars are used, like "DISABLE_TEST_MERGE", where we just check for the nullptr when the envvar is not declared to go with the default behaviour of the software. In this case the TIGERA_PROXY will be always Disabled except when explicitly enabled.

Reply:
It feels ambiguous to see what is effectively `TIGERA_TPROXY_ENABLED=""`, I tried to take a look at `DISABLE_TEST_MERGE` and couldn't find it. I also tried to find envoy environment variables to see how they use environment variables to see if there was a significant amount of "prior art" with envoy environment variables. Could you share where these environment variables are defined/used?

Reply:
```suggestion
		{Name: "TIGERA_TPROXY", Value: "Enabled"},
```